ldap/servers/slapd/back-ldbm/db-mdb/mdb_rdncache.c

```diff
@@ -351,7 +350,6 @@ rdncache_new_elem(RDNcacheHead_t *head, ID entryid, ID parentid, int nrdnlen,
     }
     head->head_per_id[idx_by_id] = elem;
     head->head_per_rdn[idx_by_rdn] = elem;
-    idx_by_id = rdncache_lookup_by_id(head, parentid);
     elem->head = head;
```

Comment:
Just curious, what is this fixing?  I wonder if this is related to import crash I found...

Reply:
I do not think iy is related to the crash.  It fix a variable set but not used warning ( idx_by_id is not used afterwards so there is no need to compute it ... )